dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -160,4 +160,8 @@ compression.duplicateOverwriteAndAdd.useraction=If the default configuration mus
 
 compression.unsupportedAlgorithm=CWWKT0033W: An attempt was made to configure {0} as the server-preferred algorithm. The supported values are gzip, x-gzip, zlib, deflate, identity, and none. The default configuration {1} is used instead.
 compression.unsupportedAlgorithm.explanation=The configuration is attempting to set an unsupported algorithm as the server-preferred algorithm to use for compression. 
-compression.unsupportedAlgorithm.useraction= When defining the value, ensure that it is set to gzip, x-gzip, deflate, zlib, identity, or none.
\ No newline at end of file
+compression.unsupportedAlgorithm.useraction= When defining the value, ensure that it is set to gzip, x-gzip, deflate, zlib, identity, or none.
+
+decompression.tolerance.reached=CWWKT0034E: Attempted to decompress a body with a ratio larger than tolerated.
+decompression.tolerance.reached.explanation=The amount of tolerated decompression cycles, defined by the 'decompressionTolerance' httpOption attribute, which resulted in a decompression ratio larger than the configured limit, has been reached. The decompression limit is defined by the 'decompressionRatioLimit' httpOption attribute.
```

Comment:
Change,
"The amount of tolerated decompression cycles, defined by the 'decompressionTolerance' httpOption attribute, which resulted in a decompression ratio larger than the configured limit, has been reached. The decompression limit is defined by the 'decompressionRatioLimit' httpOption attribute."
to,
"The maximum number of decompression cycles has reached, resulting in a decompression ratio larger than the configured limit. The decompression limit is defined by the 'decompressionRatioLimit' httpOption attribute."

Reply:
Can we word this as, "The maximum number of decompression cycles resulting in a decompression ratio larger than the configured limit has been reached. The decompression limit is defined by the 'decompressionRatioLimit' httpOption attribute."